app/views/bikes/edit_alert_purchase.html.haml

```diff
@@ -11,12 +11,18 @@
               -# highlight the same nav menu item
               = @edit_templates.select { |k, v| @edit_template.match(/#{k}/) }.values.first
 
+          - currency = t(I18n.locale, scope: %i[money currencies])
```

Comment:
I think it might make sense to put this into a helper, a la `theft_alert_plan_title`.

Reply:
Good call thanks -- updated in 1d39916e